xbmc/video/VideoDatabase.h

```diff
@@ -1000,7 +1000,9 @@ class CVideoDatabase : public CDatabase
   \return The dbId of the season.
   */
   int AddSeason(int showID, int season, const std::string& name = "");
-  int AddSet(const std::string& strSet, const std::string& strOverview = "");
+  int AddSet(const std::string& strSet,
+             const std::string& strOverview = "",
+             const bool& updateOverview = true);
```

Comment:
It does not make much sense to pass simple data types by reference. Please change the type to a simple `bool`, passed by value.

Reply:
Done